### PR TITLE
Dockerized deployment and localization updates

### PR DESCRIPTION
**Pull request by peterzen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,29 +3,41 @@ dcrweb
...
+
+#### Downloading web site code
+
+Make sure you have [git](https://git-scm.com/) installed.
 
 ```sh
-git clone https://github.com/decred/dcrweb
-cd dcrweb
-npm install
+git clone https://github.com/peterzen/dcrweb
```
Comment (dajohi) [inline README.md]: should revert this url to decred/dcrweb

Comment (peterzen) [inline README.md]: thanks, fixed

**Comment by peterzen** [comment]
Does this look OK now?

**Comment by dajohi** [comment]
Was scripts.js and brand.js missed?

https://github.com/decred/dcrweb/pull/103/files

**Pull request merged**